holidays/countries/albania.py

```diff
@@ -28,69 +26,90 @@ class Albania(HolidayBase):
     """
 
     country = "AL"
-
-    def _add_holiday(self, holiday_date: date, holiday_name: str) -> date:
-        dt = holiday_date
-
-        self[dt] = holiday_name
-        if self.observed and self._is_weekend(dt):
-            dt += rd(weekday=MO(+1))
-            while dt.year == holiday_date.year and dt in self:
-                dt += td(days=+1)
-            self[dt] = f"{holiday_name} (Observed)"
-
-        return dt
+    special_holidays = {
+        2022: ((MAR, 21, "Public Holiday"),),
+    }
 
     def _populate(self, year: int) -> None:
+        def _add_with_observed(
+            hol_date: date, hol_name: str, days: int = +1
```

Comment:
I know what you mean here but it looks weird to me somehow (vs native `td()` context).